.github/workflows/scale-test-egw.yaml

```diff
@@ -192,7 +199,7 @@ jobs:
         timeout-minutes: 30
         shell: bash
         run: |
-          for image in cilium-ci operator-aws-ci hubble-relay-ci ; do
+          for image in cilium-ci operator-aws-ci ; do
```

Comment:
nit: could be swapped to:
```
      - name: Wait for images
        uses: ./.github/actions/wait-for-images
        with:
          SHA: ${{ steps.vars.outputs.sha }}
          images: cilium-ci operator-aws-ci cilium-cli-ci
```
(I've added cilium-cli-ci as well)
you could even do it as a separate step before running matrix, similar to this:
https://github.com/cilium/cilium/blob/457fdd88d9b008ce8d71892530ee5872fad48cec/.github/workflows/conformance-ingress.yaml#L83-L98